frontend/src/API.js

```diff
@@ -37,8 +37,15 @@ export const URLS = {
 export const useExperiment = (slug) =>
     useGet(API_BASE_URL + URLS.experiment.get(slug));
 
-export const useParticipant = (urlQueryString) =>
-    useGet(API_BASE_URL + URLS.participant.current + urlQueryString);
+export const getParticipant = async (urlQueryString) => {
+    try {
+        const response = await axios.get(API_BASE_URL + URLS.participant.current + urlQueryString);
+        return response.data;
+    } catch (err) {
+        console.error(err);
+        return null;
+    }
+}
```

Comment:
Perhaps it's better to leave the error-handling at the place where you can decide what to do with the error. (see next comment)

Reply:
I tried to make the code accessible elsewhere in the application -- but indeed, we don't need to call this in multiple locations, that's the whole point of using the store.